app/pings.yaml

```diff
@@ -18,3 +18,14 @@ activation:
     - https://github.com/mozilla-mobile/fenix/pull/1707#issuecomment-486972209
   notification_emails:
     - fenix-core@mozilla.com
+
+installation:
+  description: >
+    Intended for counting user installs.
```

Comment:
Can you please be more explicit about what this is, its purpose, etc.?

Reply:
Maybe someone from Glean team can provide some input regarding this?

Reply:
I think that the person/owner of this ping (who ? Data Science? The Fenix Team?) should provide a reasonable description of why we're asking our users to share this data with us: what is this for and so on :)